src/main/scala/centaur/CacheFormulas.scala

```diff
@@ -0,0 +1,37 @@
+package centaur
+
+import java.nio.file.Path
+
+import cats.implicits._
+import Operations._
+import Test.testMonad
+
+/**
+  * A collection of test formulas which can be used, building upon operations by chaining them together via a
+  * for comprehension. These assembled formulas can then be run by a client
+  */
+object CacheFormulas {
+
+  def runCachingWorkflow(request: WorkflowRequest) = {
+    for {
+      s <- TestFormulas.runWorkflowUntilTerminalStatus(request, Succeeded)
+      r <- verifyInputsOutputs(s, request)
```

Comment:
I'm all for single letter variables like `i` for short iteration loops, usually. I can even do `case (k, v)` for key/value pairs.

But in this case I would request longer variable names specifying what `s` and `r` are abbreviations for.